app/redux/app/actions.js

```diff
@@ -0,0 +1,18 @@
+export const actionType = {
+    SET_LOADING: "SET_LOADING",
+    SET_MESSAGE: "SET_MESSAGE",
+}
+
+export const setLoading = (isLoading) => {
+    return {
+        type: actionType.SET_LOADING,
+        isLoading: isLoading,
+    }
+}
+
+export const setMessage = (message) => {
+    return {
+        type: actionType.SET_MESSAGE,
+        message: message,
+    }
+}
```

Comment:
will be better if components will have their own reducer and action file because it will be easier to work and make fixes in the future instead of storing all in one file

Reply:
But we have separate actions, reducers for each component as well.
For now, we have two components for user and app.